resources/mail/layouts/html.php

```diff
@@ -1 +1,20 @@
+<?php
+/* @var $content string Mail contents as view render result */
+?>
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml">
+<head>
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+    <title>MAIL_VIEW_TITLE</title>
+</head>
+<body>
+
 <?= $content ?>
+
+<footer style="margin-top: 5em">
+-- <br>
+APP_NAME
```

Comment:
?

Reply:
In order to make it better we should print app name, view title and character encoding but I don’t know how to make it in Yii3.

Yii2 equivalents:
* `<?= Html::encode(Yii::$app->name) ?>`
* `<?= Html::encode($this->title) ?>`
* `<?= Yii::$app->charset ?>`

Reply:
Set encoding to UTF 8. There's no application title in Yii 3 (yet).

Reply:
Encoding already set, the title can be leaved constant for now or removed at all, as it isn’t displayed anywhere in mail agents.

Reply:
Let's remove it.

Reply:
Done.